conf/app.routes

```diff
@@ -25,14 +25,14 @@ GET         /unauthorised
 GET         /:taxYear/task-list                                                     controllers.journeys.TaskListController.onPageLoad(taxYear: TaxYear)
 GET         /:taxYear/task-list-prepop                                              controllers.journeys.PrepopTaskListController.onPageLoad(taxYear: TaxYear)
 
-GET         /:taxYear/:businessId/:journey/details/completed-section                controllers.journeys.SectionCompletedStateController.onPageLoad(taxYear: TaxYear, businessId: BusinessId, journey: String, mode: Mode = NormalMode)
```

Comment:
- I feel 'details/` is superfluous, happy to revert if you disagree.

- As businessId = journey in business-agnostic journeys e.g. trade-details and NICs, we may need to make exceptions for those journeys to display a clean URL. I have added a bullet point to that effect in the tech debt tracker.

Reply:
These URLs were written from the documentation, but I agree they could be better so maybe @AngelicaScholts could give her thoughts?
https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=SASUB&title=Self-employment+UX